.github/workflows/changeset.yml

```diff
@@ -0,0 +1,18 @@
+name: Changeset
+
+on:
+  pull_request:
```

Comment:
Initially, thought to run workflow only if .changeset/** changes but then realized that package.json of packages may change too, causing changeset validation to fail. So now including all paths, although could still narrow down to .changeset/** and packages/**/package.json?

Reply:
sounds reasonable to run as-needed (just for changeset + package.json changes) although may also need to consider the `pnpm-workspace.yaml` and ~any `package.json` (e.g. `**/package.json`)

but also running it for every commit is fine too, should be lightweight enough to not matter much

Reply:
Perhaps then best to not restrict any paths since it is indeed lightweight, and don't want to miss anything accidentally.